app/services/ingest_iati_activities.rb

```diff
@@ -305,6 +306,21 @@ def call
     new_activity.gdi = "Replace me"
   end
 
+  private def add_fstc_applies(new_activity:)
+    new_activity.fstc_applies = case new_activity.aid_type
+    when "C01"
+      true
+    when "E01"
+      true
+    when "D01"
+      true
+    when "GO1"
+      false
+    else
+      false
```

Comment:
I agree, this will keep the test suite happy, we shouldn't be using the ingest code again.